themes/digital.gov/src/scss/new/home/_communities.scss

```diff
@@ -34,3 +34,13 @@ $communities-of-practice: (
     }
   }
 }
+
+.usa-card__footer {
+  a.usa-button {
+    @include u-display('flex');
+    @include u-flex('align-center');
+    @include u-flex('justify-center');
+    @include u-minh(8);
+    @include u-margin-right(0);
```

Comment:
It doesn't look like we need this margin-right.

Reply:
Good call, probably here because of some browser caching issues while testing.